lib/Doctrine/ORM/Mapping/Driver/AnnotationDriver.php

```diff
@@ -327,10 +327,6 @@ public function loadMetadataForClass($className, ClassMetadata $metadata)
             // @Column, @OneToOne, @OneToMany, @ManyToOne, @ManyToMany
             $columnAnnot = $this->reader->getPropertyAnnotation($property, Mapping\Column::class);
             if ($columnAnnot) {
-                if ($columnAnnot->type === null) {
-                    throw MappingException::propertyTypeIsRequired($className, $property->getName());
```

Comment:
is this method still used anywhere else? otherwise we can `@deprecated` it in `MappingException`.

Reply:
It seems it is not. I have added `@deprecated`